userinterface/driveui.cpp

```diff
@@ -177,3 +176,12 @@ void DriveUI::on_apSetActiveIDButton_clicked()
                 mCurrentVehicleConnection->setActiveAutopilotID(apID);
         }
 }
+
+void DriveUI::on_vehicleParameterButton_clicked()
+{
+    if (mVehicleParameterUI.isNull())
+        mVehicleParameterUI = QSharedPointer<VehicleParameterUI>::create(this);
+    mVehicleParameterUI->setCurrentVehicleConnection(mCurrentVehicleConnection);
+    mVehicleParameterUI->show();
+    this->releaseKeyboard();
```

Comment:
Once VehicleParameterUI is hidden, DriveUI should grab the keyboard again (but only if DriveUI still visible), right? We might need to rework the whole keyboard grabbing thing, so no priority for this PR.

Reply:
Leaving this for now